core/meta/src/TClass.cxx

```diff
@@ -1531,7 +1533,9 @@ void TClass::Init(const char *name, Version_t cversion,
       } else {
          // In this case we initialised this TClass instance starting from the fwd declared state
          // and we know we have no dictionary: no need to warn
-         ::Warning("TClass::Init", "no dictionary for class %s is available", fName.Data());
+         const bool ispairbase = TClassEdit::IsStdPairBase(fName.Data()) && !IsFromRootCling();
```

Comment:
> What about other implementations not using this?

Because it is not worth making an exhaustive search, it is "just" removing/hiding an information-less warning message.  If there are other reports about the other pair's base class then we can add it.

Reply:
> ⚠️ ⚠️ ⚠️ The overload IsStdPairBase(ROOT::Internal::TStringView name) seems to call the wrong function!

You are right.  Fixed.